models/tsodyks2_connection.h

```diff
@@ -60,8 +60,8 @@ Parameters
 The following parameters can be set in the status dictionary:
 
 ========  ======  ========================================================
- U        real    Maximum probability of release (U1) [0,1], default=0.5
- u        real    Maximum probability of release (U_se) [0,1],
+ U        real    Parameter determining the increase in u with each spike (U1) [0,1], default=0.5
```

Comment:
"tsodyks2" seems to correspond to another paper than "tsodyks". The description in the paper for tsodyks2 (Maass & Markram 2002) for U is "the fraction of synaptic resources that are used for a single spike", and u is described as "how much of the fraction of synaptic efficacy is actually used for a spike". Could you confirm?

Reply:
Hi, thanks for replying. "tsodyks2_connection" is also linked to the first 2 reference articles (Tsodyks, Markram, 1997; Fuhrman, Segev, Markram, Tsodyks, 2002). Combining these 2 articles, the modeling of u (as U_SE in the 1st and 2nd article) and U (as U1 second article) is the same as in (Tsodyks, Uziel, Markram, 2000). In these articles, u stands for the instantaneous release probability (or the fraction of available synaptic efficacy that is actually used), and U is the constant for modifying u with each spike.

Regarding the 3rd article (Maass & Markram 2002), it can be seen that u and U there are also compatible to the above. The description for u "how much of the fraction R_n+1 of the synaptic efficacy is actually used for the (n+1)th spike" is the same as "The probability of release", because the actually released fraction of synaptic efficacy in a spike is u_n*R_n, where R_n is the available fraction, and u_n is the released fraction of R_n. 

The explanation for U "the fraction of synaptic resources that are used for a single spike" means that it determine the value of u in the case of only 1 spike or 1st spike if there are many (which is why it is named U1 in the 2nd article). However, this U also defines the step increase in u, according to the models in all these articles, including that in the 3rd article: "The factor u_n+1... It is increased from u_n to U(1-u_n) + u_n". I think this point is more important for a simulation, hence I made the pull request thereby.  

I hope this is a clear explanation. If there’s further question or suggestion regarding this, please kindly let me know.